src/Compilers/Shared/RuntimeHostInfo.cs

```diff
@@ -53,15 +53,15 @@ internal static (string processFilePath, string commandLineArguments, string too
         /// Get the path to the dotnet executable. In the case the host did not provide this information
         /// in the environment this will return simply "dotnet".
         /// </summary>
+        /// <remarks>
+        /// See the following issue for rationale why only %PATH% is considered
+        /// https://github.com/dotnet/runtime/issues/88754
```

Comment:
I'd honestly prefer the docs link here, that was far clearer than this thread :slightly_smiling_face: